init.lua

```diff
@@ -187,7 +226,6 @@ function minetest.register_chatcommand(name, def)
 	return old_register_chatcommand(name, def)
 end
 
-
```

Comment:
try and avoid cleaning up unrelated code at the same time as writing significant changes.